library/core/src/iter/sources.rs

```diff
@@ -102,10 +101,8 @@ pub fn repeat<T: Clone>(elt: T) -> Repeat<T> {
 /// An iterator that repeats elements of type `A` endlessly by
 /// applying the provided closure `F: FnMut() -> A`.
 ///
-/// This `struct` is created by the [`repeat_with`] function.
+/// This `struct` is created by [`repeat_with`].
```

Comment:
```suggestion
/// This `struct` is created by [`repeat_with()`].
```
and same newline nit as above